gateway-service-impl/src/main/java/org/hypertrace/gateway/service/common/converters/QueryAndGatewayDtoConverter.java

```diff
@@ -417,8 +417,8 @@ public static Filter convertToQueryFilter(
   }
 
   public static Filter addTimeAndSpaceFiltersAndConvertToQueryFilter(
-      long startTimeMillis,
-      long endTimeMillis,
+      long startTime,
```

Comment:
I don't believe this util cares about units, but just to be aware, others do (anything using datetimeconvert). We'll probably want to lookup the timestamp unit when we get to that point.